### PR TITLE
Ee merge

### LINKED ISSUE #818: Forms feature in open source
By any chance do we get to see forms feature in open source botfront ?

At least the minimum feature of forms and slots, required field and validation would be great to see in open source.

I am currently looking for such features, found few but they are outdated and not ready to use with recent version of RASA.

### LINKED ISSUE #870: How can I buy an Enterprise?
How can I buy an Enterprise version?

### PR DESCRIPTION
**Pull request by MatthieuJnon**
Closes #870, closes #818.

### CODE REVIEW

**Comment by pheel** [comment]
@MatthieuJnon Squashing this sounds a little extreme, since we're importing _master_ history... If we do that might as well squash our CE master history.

**Comment by MatthieuJnon** [comment]
@pheel, when ready, I'm thinking we can just rename it master and force push it on top ?

**Comment by pheel** [comment]
Oh yeah that makes sense.

File: api/.babelrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (pheel) [inline api/.babelrc:1]: Should the api be in the repo?

File: botfront/package.json [hunk 2]
```diff
@@ -1,14 +1,17 @@
 {
   "name": "botfront",
-  "version": "1.0.0-rc.3",
+  "version": "1.0.0-ee.1-rc.3",
```
Comment (pheel) [inline botfront/package.json:3]: No `ee`s

File: .github/workflows/delete-artifacts.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: 'Delete old artifacts'
```
Comment (pheel) [inline .github/workflows/delete-artifacts.yml:1]: no longer needed

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -148,8 +148,12 @@ jobs:
     if: github.event_name == 'push' && (contains(github.ref, 'master') || contains(github.ref, 'tags'))
     needs: [unit-bf, build-bf, integration]
     steps:
-      - name: log in
-        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
```
Comment (pheel) [inline .github/workflows/main.yml:151]: Use the one on master
https://github.com/botfront/botfront/blob/master/.github/workflows/main.yml

**Review by pheel — APPROVED** [review]

**Pull request merged**